samples/cloudwatch/README.md

```diff
@@ -2,52 +2,75 @@
 
 This sample describes how to configure Amazon CloudWatch to collect metrics
 from the services and components in SAS Viya. Complete the deployment
-and configuration steps in this document to enable Amazon CloudWatch to collect and
-visualize metric information from both SAS Viya as well as the monitoring components deployed from this repository. This sample currently does not provide information for using Amazon applications to view SAS Viya log messages.
+and configuration steps in this document to enable Amazon CloudWatch to collect
+and visualize metric information from both SAS Viya as well as the monitoring
+components deployed from this repository. This sample currently does not
+provide information for using Amazon applications to view SAS Viya log
+messages.
 
 ## CloudWatch Agent
 
-AWS uses the CloudWatch agent to collect metrics. There are two types of CloudWatch agents that you might want to use in your environment:
-
-- The CloudWatch agent with Prometheus monitoring scrapes metrics from Prometheus sources, such as SAS Viya. The metrics can then be converted and mapped for display in CloudWatch. Because SAS Viya monitoring uses Prometheus, this sample explains how to deploy and configure only the CloudWatch agent for use with Prometheus. 
-
-- The default CloudWatch agent collects metrics from AWS nodes. Although these metrics are not specific to SAS Viya, you might want to use both agents so you can obtain a complete view of your environment's performance. For information about using the default agent, see the 
-[CloudWatch documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Install-CloudWatch-Agent.html). If you do use the default agent, you must add tolerations to the
-agent's daemonset definition so that the agent can run on nodes that are tainted to support SAS Viya workload node placement. See [CloudWatch Agent Daemonset and Workload Node Placement](#wnp_tolerations).
+AWS uses the CloudWatch agent to collect metrics. There are two types of
+CloudWatch agents that you might want to use in your environment:
+
+- The CloudWatch agent with Prometheus monitoring scrapes metrics from
+Prometheus sources, such as SAS Viya. The metrics can then be converted and
+mapped for display in CloudWatch. Because SAS Viya monitoring uses
+Prometheus, this sample explains how to deploy and configure only the
```

Comment:
I think this last sentence needs to be more prominent or broken out as its own paragraph; it is too easily missed otherwise.

Reply:
I moved the last sentence to a new paragraph as mentioned

Reply:
Thanks, I think that's more clear.  I think it might be even clearer if we move it to after the 2nd bullet point and explicitly state that this sample focuses on the first agent and not the second.  

Something like:
_Because SAS Viya monitoring uses Prometheus, this sample explains only how to deploy and configure the CloudWatch agent for use with Prometheus and does not discuss the default CloudWatch agent in great detail._

Reply:
Done